src/hid.c

```diff
@@ -148,6 +148,36 @@ fido_dev_info_ptr(const fido_dev_info_t *devlist, size_t i)
 	return (&devlist[i]);
 }
 
+int
+fido_dev_info_set(fido_dev_info_t *devlist, size_t i,
```

Comment:
Suggest renaming to `fido_dev_set_info()` for consistency with other setters in `libfido2`.

Reply:
The name `fido_dev_set_info` suggests to me that it operates on a `fido_dev_t`, but that's not what this does -- it operates on a `fido_dev_info_t` array, like `fido_dev_info_new` returns.  Maybe `fido_dev_info_set_parameters` or `fido_dev_info_set_entry` or `fido_dev_info_init`, if `fido_dev_info_set` still seems wrong?

Reply:
You are absolutely correct; the current name is fine by me. :+1: